src/Validators/Validator.Strings.cs

```diff
@@ -109,4 +109,22 @@ internal static void ThrowIfLengthNotEquals(string value, string paramName, Exce
             ExceptionThrower.Throw(paramName, exceptionCustomizations, $"String length should be equal to {length}.");
         }
     }
+
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    internal static void ThrowIfEndsWith(string value, string paramName, ExceptionCustomizations? exceptionCustomizations, string str)
+    {
+        if (value.EndsWith(str))
+        {
+            ExceptionThrower.Throw(paramName, exceptionCustomizations, $"String should not end with {str}.");
```

Comment:
Small change to align with the other error messages that include the given string (H&B)
```suggestion
            ExceptionThrower.Throw(paramName, exceptionCustomizations, $"String should not end with '{str}'.");
```